ios/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -94,6 +94,10 @@ - (id)init
                                                  selector:@selector(headphoneConnectionDidChange:)
                                                      name:AVAudioSessionRouteChangeNotification
                                                    object: [AVAudioSession sharedInstance]];
+        [[NSNotificationCenter defaultCenter] addObserver:self
+                                                 selector:@selector(brightnessDidChange:)
+                                                     name:UIScreenBrightnessDidChangeNotification
+                                                   object: nil];
 #endif
```

Comment:
I fail to understand how how *this* symbol is available on tvOS and the `.brightness` one on UIScreen is not, but...it is on tvOS!
https://developer.apple.com/documentation/uikit/uiscreenbrightnessdidchangenotification 😆 

But of course, there is no actual way to *get* the brightness (or alter it), so...whatever Apple